grammar.js

```diff
@@ -345,12 +345,12 @@ module.exports = grammar({
         ),
 
         string_literal: $ => seq(
-            alias(/b?"/, '"'),
+            token('"'),
             repeat(choice(
                 $.escape_sequence,
-                // $._string_content
+                /[^\\]/,
```

Comment:
Extracting this into it's own rule does not seem feasible, as the regex for the string content depends on the type of string we are parsing. This one does not allow any unescaped backslashes while e.g. single-quote strings don't allow single-quotes.